corehq/apps/geospatial/static/geospatial/js/case_grouping_map.js

```diff
@@ -17,11 +17,13 @@ hqDefine("geospatial/js/case_grouping_map",[
         Visible: 'visible',
     };
 
-    const DEFAULT_MARKER_COLOR = "rgba(128,128,128,0.25)";
```

Comment:
This should not be. Have you rebased, @SmittieC ?

Reply:
No, but I'll merge master into this branch, since you merged you branch

Reply:
Actually, this is not needed anymore. The function was updated to only show markers _with_ groups

Reply:
Update: This is needed, but @Charl1996 will reintroduce this on a branch he's working on. Not a biggy. I'm resolving this convo